app/models/tasks.js

```diff
@@ -59,5 +52,27 @@ module.exports = function (sequelize, DataTypes) {
       )
     );
   };
+
+  Tasks.inserProjectTaskDescription = async = (tasks, projectId) => {
```

Comment:
@iamhassanali This is not right... there is an extra `=` after `async`. Please remove it.